### PR TITLE
completed javascript-loops

### PR DESCRIPTION
**Pull request by bug-smith**
Completed.
Screenshot 2023-08-31 at 8 04 06 AM

### CODE REVIEW

File: javascript-loops/javascript-loops-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- _When_ does the **condition** expression of a `for` loop get evaluated?
+
+- _When_ does the **final** expression of a `for` loop get evaluated?
+
+- Besides a `return` statement, which exits its entire function block, which keyword exits a loop before its **condition** expression evaluates to `false`?
+
+- What does the `++` increment operator do?
+
+- How do you iterate through the keys of an object?
+
```
Comment (shawnkost) [inline javascript-loops/javascript-loops-quiz-notes.md]: Don't forget to answer the quiz questions

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  }
+  return evenNumbers;
+}
+
+getEvenNumbersToTwenty();
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (count < times) {
```
Comment (shawnkost) [inline javascript-loops/main.js]: This should be less than or equal to. You're currently doing 1 less than `times`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple tweaks here Bug!

File: javascript-loops/main.js (hunk 2 above)
Comment (bug-smith) [reply javascript-loops/main.js]: ahhh yes that makes total sense as count starts at 1, thank you sir!

the sneaky undefined has been destroyed!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**